src/oauth.js

```diff
@@ -0,0 +1,60 @@
+const axios = require('axios');
+const Koa = require('koa');
+const Router = require('koa-router');
+const manifest = require('../manifest.json');
+
+const app = new Koa();
+const router = new Router();
+
+const TS_URL_BASE = 'https://api-smoketest.eu-west-1.test.ts.sv';
```

Comment:
so maybe make it also from `TS_API_HOST` in the .env file?

Reply:
yeah. i just thought the original transformation on the value from the .env file wasn't very clear:

https://github.com/Tradeshift/bucko/blob/42755de6e31c2523f88c052cd082fb9fdea5aed7/src/auth.js#L45

Reply:
I think the original one looks okay as well :)